Player.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+
+
+namespace Card_Game
+{
+    public class Player 
+    {
+        bool isPlayerInHand = false;  // na true, jeśli prowadzi grę
+        bool isPlayerInGame = false; // na true, jeśli jeszcze w grze
+        List<Card> Cards {get; set;}
```

Comment:
How about ff Player has a PlayerDeck instead of Cards? Then we could use deck's methods in it.

Reply:
thx